nd4j/src/main/scala/nexus/impl/nd4j/UntypedFloatTensor.scala

```diff
@@ -0,0 +1,53 @@
+package nexus.impl.nd4j
+
+import org.nd4j.linalg.api.ndarray.INDArray
+
+
+trait UntypedFloatTensor { self =>
```

Comment:
This would mostly be useless -- directly using INDArray reduces one layer of wrapping.